File: packages/core/src/paths.ts
import os from "node:os";
import path from "node:path";

/** Where the local skilljit catalog db lives: ~/.skilljit/catalog.db. */
export function defaultCatalogPath(): string {
  const home = process.env.SKILLJIT_HOME ?? os.homedir();
  return path.join(home, ".skilljit", "catalog.db");
}

File: python/src/skilljit/paths.py
"""Where the local skilljit catalog db lives — must match the Node CLI's
default exactly, since both read/write the same file."""

import os
from pathlib import Path


def default_catalog_path() -> Path:
    home = Path(os.environ.get("SKILLJIT_HOME", Path.home()))
    return home / ".skilljit" / "catalog.db"
